JitsiConferenceEvents.js

```diff
@@ -133,6 +133,11 @@ export const PARTICIPANT_PROPERTY_CHANGED
     = 'conference.participant_property_changed';
 
 /**
+ * Indicates that the conference has switched between JVB and P2P connections.
+ */
+export const P2P_STATUS = 'conference.p2pConnectionStatus';
```

Comment:
We refer to that as sessions only inside of JitsiConference. In the outside world we care only whether we're running P2P or not. I will rename the string.